editor/editor_file_system.cpp

```diff
@@ -1787,7 +1787,7 @@ void EditorFileSystem::_reimport_file(const String &p_file) {
 	//as import is complete, save the .import file
 
 	FileAccess *f = FileAccess::open(p_file + ".import", FileAccess::WRITE);
-	ERR_FAIL_COND(!f);
+	ERR_FAIL_COND_MSG(!f, "Cannot open file from path: " + p_file + ".import .");
```

Comment:
Use quotes, then you don't need the awkward space before "." (I'd argue it's not needed even without quotes, but I understand why you wanted to disambiguate).

Reply:
Several occurrences above and below.